tpm/encoding.go

```diff
@@ -140,6 +140,25 @@ func packType(buf io.Writer, elts []interface{}) error {
 	return nil
 }
 
+func unpackKeyHandleList(b []byte) ([]Handle, error) {
+	// TODO(kwalsh): handle pack/unpack of TPM_KEY_HANDLE_LIST more gracefully
+	buf := bytes.NewBuffer(b)
+	var n uint16
+	if err := unpackType(buf, []interface{}{&n}); err != nil {
+		return nil, err
+	}
+	if n == 0 {
+		return nil, nil
+	}
+	h := make([]Handle, int(n))
```

Comment:
I believe that make accepts any integer type in its size argument, so I don't think you need to cast to int here.